### PR TITLE
List ms teams users

### PR DESCRIPTION
**Pull request by VladyslavBanul**
https://app.asana.com/0/1201128813141482/1201275099482155/f
https://app.asana.com/0/1201128813141482/1201275099482153/f

### CODE REVIEW

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/OtherLicenses.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import IconButton from "@material-ui/core/IconButton";
+// import IconButton from "components/common/Form/IconButton";
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/OtherLicenses.tsx:2]: You need this import ?

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/OtherLicenses.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import { EyeClosed, EyeOpened, Minus } from "components/Icons";
+import { FC, useState } from "react";
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/OtherLicenses.tsx:4]: move this import to libs please

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+
+import { TableData, TableProps } from "utils/types/tableConfig";
+
+import Table from "components/Table";
+import OtherLicenses from "./components/OtherLicenses";
+import FormSelect from "components/common/Form/FormSelect";
+import AssignedNumber from "./components/AssignedNumber";
+
+import useStyles from "./styles";
+import { yupResolver } from "@hookform/resolvers/yup";
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx:19]: move this import to libs please

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+import { object, string } from "yup";
+import TableSelectedRowsStore from "storage/singletons/TableSelectedRows";
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx:21]: move this import to storage please

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+import { TableData, TableProps } from "utils/types/tableConfig";
+
+import Table from "components/Table";
+import OtherLicenses from "./components/OtherLicenses";
+import FormSelect from "components/common/Form/FormSelect";
+import AssignedNumber from "./components/AssignedNumber";
+
+import useStyles from "./styles";
+import { yupResolver } from "@hookform/resolvers/yup";
+import { object, string } from "yup";
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx:20]: move this import to libs please

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx [hunk 6]
```diff
@@ -0,0 +1,137 @@
...
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [t],
+  );
+
+  const onSubmit = (values: { assignedNumber: string }) => {
+    editMsTeamsUserNumber(
+      tenantID,
+      subscriptionID,
+      TableSelectedRowsStore.selectedRowsValues[0].original.msTeams.id,
+      values.assignedNumber === "Unselect number"
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/index.tsx:115]: Could you please add locale if it possible ?

File: src/levels/subscription/components/MsTeams/index.tsx [hunk 7]
```diff
@@ -10,8 +10,15 @@ import Tabs from "components/Tabs";
 import Loader from "components/Loader/Loader";
 import O365Tenant from "./components/O365Tanant";
 import O365Admin from "./components/O365Admin";
+import { t } from "services/Translation";
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/index.tsx:13]: move this import to services / utils  please

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/AssignedNumber.tsx [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  const classes = useStyles();
+  return draasUserInfo && draasUserInfo?.phoneNumber ? (
+    <div className={classes.numberWithLabel}>
+      <div>{draasUserInfo.phoneNumber}</div>
+      <div className={clsx([classes.label], [classes[draasUserInfo.status]])}>
+        {draasUserInfo.status}
+      </div>
+    </div>
+  ) : (
+    <div>no phonenumber</div>
```
Comment (SergeyKoliadintsev) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/AssignedNumber.tsx:21]: add locale please

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/AssignedNumber.tsx [hunk 9]
```diff
@@ -0,0 +1,27 @@
+import clsx from "clsx";
+import { FC } from "react";
+import { useTranslation } from "react-i18next";
+
+import useStyles from "../styles";
+
+const AssignedNumber: FC<{
+  draasUserInfo: {
+    [key: string]: any;
+    status: "active" | "updating" | "error";
```
Comment (viktorsmakhtin) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/components/AssignedNumber.tsx:10]: need to provide a locale for this

File: src/levels/subscription/components/MsTeams/components/MsTeamsUsers/styles.ts [hunk 10]
```diff
@@ -0,0 +1,83 @@
+import { makeStyles } from "@material-ui/core";
+import { ThemeDefaultOptions } from "utils/types/themeConfig";
+//@ts-ignore
```
Comment (viktorsmakhtin) [inline src/levels/subscription/components/MsTeams/components/MsTeamsUsers/styles.ts:3]: pls delete it

**Pull request merged**